mlflow/transformers/__init__.py

```diff
@@ -2761,6 +2802,32 @@ def is_uri(s):
                 error_code=BAD_REQUEST,
             )
 
+    def _wrap_input_in_prompt_template(self, input_data):
+        """
+        Wraps the input data in the specified prompt template. If no template is
+        specified, or if the pipeline is an unsupported type, or if the input type
+        is not a string or list of strings, then the input data is returned unchanged.
```

Comment:
should i throw instead? i felt like that would be risky in case *somehow* an incompatible model gets saved with a template.

Reply:
+1 to throw (or at least warning) - it's hard to debug why template is dismissed.

Reply:
+1 to throw here as well. It's unintended behavior and is the equivalent of us messing with input data values in any other type of model.